contracts/contracts/GrantRoundManager.sol

```diff
@@ -18,22 +18,16 @@ contract GrantRoundManager {
   /// @notice Address of the GrantRegistry
   GrantRegistry public immutable registry;
 
-  /// @notice Address of the Uniswap V3 Router used for token swaps
-  ISwapRouter public immutable router;
-
   /// @notice Address of the ERC20 token in which donations are made
   IERC20 public immutable donationToken;
 
-  /// @notice Used during donations to temporarily store swap output amounts
   mapping(IERC20 => uint256) internal swapOutputs;
 
-  /// @notice Used during donations to temporarily store donation ratio sums to ensure totals are always 100%
   mapping(IERC20 => uint256) internal donationRatios;
 
   /// @notice WETH address
-  IERC20 public constant WETH = IERC20(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+  IERC20 public WETH;
```

Comment:
Let's get rid of this WETH and just use the `WETH9` that's already part of the `SwapRouter`. Currently the contract has `IERC20 WETH` and `address WETH9` which is a bit redundant/unnecessary

This will also require us to change `WETH` to `IERC20(WETH9)` in other places